src/app/inner-pages/shared/components/job-full/job-full.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { getJobs, State } from 'app/core/reducers';
+import { Job } from 'app/shared/models';
+import { JobsService } from 'app/shared/services';
+import { Subscription } from 'rxjs';
+import { flatMap } from 'rxjs/operators';
```

Comment:
порядок импортов, внешниеи внутренние импорты разделены пустой строкой
```suggestion
import { Component, OnDestroy, OnInit } from '@angular/core';
import { ActivatedRoute } from '@angular/router';
import { Store } from '@ngrx/store';
import { Subscription } from 'rxjs';
import { flatMap } from 'rxjs/operators';

import { getJobs, State } from 'app/core/reducers';
import { Job } from 'app/shared/models';
import { JobsService } from 'app/shared/services';
```